components/ui/project-form.tsx

```diff
@@ -43,50 +45,48 @@ export default function ProjectForm({ defaultValues }: Props) {
       const modifiedData = {
         ...data,
         id: defaultValues?.id,
-      }
+      };
       editProject(modifiedData)
-      .catch(err => console.error('project edit error'))
-      .then(() => console.log('successfully edited'))
-      . catch(() => console.log('promise catched'));
-    } else{
+        .catch(() => console.error('project edit error'))
+        .then(() => console.log('successfully edited'))
+        .catch(() => console.log('promise catched'));
```

Comment:
What does the last catch catch? Also why catch here at all - we are not handling the error, instead we are silently ignoring the error message and logging generic one instead